ui/raidboss/data/06-ew/ultimate/the_omega_protocol.txt

```diff
@@ -236,47 +243,52 @@ hideall "--sync--"
 882.6 "Hello, Distant World 3" #sync / 1[56]:[^:]*:Omega-F:8111:/
 
 # Omega Version
-# TODO: Adjust targetable/untargetable
-884.6 "--targetable--"
+# TODO: Account for patterns
+885.3 "--targetable--"
+887.3 "--sync--" sync / 1[56]:[^:]*:Omega-M:7C02:/
 894.3 "Solar Ray 1" sync / 1[56]:[^:]*:Omega-M:81AD:/
 897.4 "Solar Ray 2" sync / 1[56]:[^:]*:Omega-M:7B02:/
 904.6 "--sync--" sync / 1[56]:[^:]*:Omega-M:7B43:/
-910.8 "Run: ****mi* (Omega Version)" sync / 1[56]:[^:]*:Omega-M:8015:/
-929.2 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B79:/
-933.3 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B79:/
-936.0 "--untargetable--"
-
-946.0 "Hello, Distant World 1" sync / 1[56]:[^:]*:Omega-F:8110:/
-946.0 "Hello, Near World 1" sync / 1[56]:[^:]*:Omega-F:7B89:/
-946.4 "Oversampled Wave Cannon" sync / 1[56]:[^:]*:Omega:7B6D:/
-947.0 "Hello, Distant World 2" #sync / 1[56]:[^:]*:Omega-F:8111:/
-947.0 "Hello, Near World 2" #sync / 1[56]:[^:]*:Omega-F:7B8A:/
-948.0 "Hello, Distant World 3" #sync / 1[56]:[^:]*:Omega-F:8111:/
-948.0 "Hello, Near World 3" #sync / 1[56]:[^:]*:Omega-F:7B8A:/
-
-962.2 "Blaster x2" sync / 1[56]:[^:]*:Omega-F:7E75:/
-964.0 "Hello, Distant World 1" sync / 1[56]:[^:]*:Omega-F:8110:/
-964.0 "Hello, Near World 1" sync / 1[56]:[^:]*:Omega-F:7B89:/
-965.0 "Hello, Distant World 2" #sync / 1[56]:[^:]*:Omega-F:8111:/
-965.0 "Hello, Near World 2" #sync / 1[56]:[^:]*:Omega-F:7B8A:/
-966.0 "Hello, Distant World 3" #sync / 1[56]:[^:]*:Omega-F:8111:/
-966.0 "Hello, Near World 3" #sync / 1[56]:[^:]*:Omega-F:7B8A:/
-
-# TODO: Adjust targetable/untargetable
-968.5 "--targetable--"
-977.4 "Solar Ray 1" sync / 1[56]:[^:]*:Omega-M:81AD:/
-980.5 "Solar Ray 2" sync / 1[56]:[^:]*:Omega-M:7B02:/
+910.7 "Run: ****mi* (Omega Version)" sync / 1[56]:[^:]*:Omega-M:8015:/
+927.8 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B9C:/
+928.8 "Superliminal Steel" sync / 1[56]:[^:]*:Omega-F:7B2A:/
+928.8 "Beyond Strength" sync / 1[56]:[^:]*:Omega-M:7B25:/
+928.9 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B79:/
+931.9 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B77:/
+932.7 "Optimized Blizzard III" sync / 1[56]:[^:]*:Omega-F:7B2D:/
+932.7 "Beyond Strength" sync / 1[56]:[^:]*:Omega-M:7B25:/
+932.8 "Diffuse Wave Cannon" sync / 1[56]:[^:]*:Omega:7B79:/
```

Comment:
I could see an argument here to not list superliminal steel / beyond strength / optimized blizzard iii / bladewhatever here, becuase it happens at the same time as 7B79 Diffuse Wave Cannon and it's really "dodge 1" and "dodge 2".

I think also there's too many Diffuse Wave Cannons here.  Probably 7B9C and 7B77 are front/back and left/right.  I think it'd be better to just leave either 7B9C/7B77 or the 7B79 ability, but not both.  There's only two attacks, not 4.

Reply:
7B79 Omega casts twice on same instance to environment, opposite facing of each other.
7B77 and 7B9C are self-cast, I think you may be right about the front/back and left/right.

I've removed 7B79, but made the 7B77|7B9C a duration 1.1. Commented out the Omega-M/F casts.